include/h2o/socketpool.h

```diff
@@ -105,6 +105,7 @@ typedef struct st_h2o_socketpool_t {
         h2o_timer_t timeout;
     } _interval_cb;
     SSL_CTX *_ssl_ctx;
+    int address_family; /* specifies required address family used for the connection. AF_UNSPEC if none (default) */
```

Comment:
```suggestion
    /**
     * specifies required address family used for the connection. AF_UNSPEC if none (default).
     */
    int address_family;
```

Maybe use a doc-comment?